tempto-core/src/main/java/io/prestodb/tempto/internal/convention/AnnotatedFileParser.java

```diff
@@ -143,11 +144,22 @@ private int findNextSectionIndex(List<String> lines)
      */
     private List<String> filterContent(List<String> contentLines)
     {
-        contentLines = contentLines.stream()
-                .filter(s -> !(isSpecialLine(s) || isBlank(s)))
-                .map(AnnotatedFileParser::unescapeLine)
-                .collect(toList());
-        return contentLines;
+        ImmutableList.Builder<String> result = ImmutableList.builder();
+
+        for (int i = 0; i < contentLines.size(); i++) {
+            String line = contentLines.get(i);
+            if (isSpecialLine(line) || isBlank(line)) {
+                continue;
+            }
+
+            while (line.endsWith(LINE_ESCAPE)) {
```

Comment:
- what if the line ends with `\\` or `\\\` ?
- instead of glueing the lines with `\<linefeed>`, can we use literal `\n` to represent a new line character in the data?